idowuadeleke/BANKA
# #165661215 refactor codebase

### What does this PR do?
refactor codebase

### Description of Task to be completed?
- refactor codebase to meet specifications

### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ch-165661215-refactor-codebase`

- new code base should meet user specifications

### Any background context you want to provide?
N/A
### What are the relevant pivotal tracker stories?
[#165661215](https://www.pivotaltracker.com/story/show/165661215)

### Screenshots (if appropriate)
N/A
